### ISSUE TITLE
Fix accessibility tests

### ISSUE THREAD

**Issue by tapina** [cambridgeweblab/common-ui#43]
Currently all the accessibility tests are failing. We need to fix these before moving to CI testing with Sauce Labs.

**Comment by jagreehal** [comment]
This will be resolved by https://github.com/cambridgeweblab/common-ui/pull/42

**Comment by norisalsaadie** [comment]
#42 is merged in. Closing issue.
